git-hub

```diff
@@ -408,15 +556,20 @@ def hub(command, args, user, comment, number, keyword, branch, sort, opened, lab
         push()
 
     elif command == "search":
-        search(keyword, user, comment, number, branch, opened, label, sort)
-
+        if(args is None):
```

Comment:
Please run `flake8` on the source code.

Reply:
I'm not sure what you wanted here. I ran flake8 and it didn't say anything about if(args is None).